### ISSUE TITLE
Indentation on if ... then ... else is all messed up

### ISSUE THREAD

**Issue by chadaustin** [andyfriesen/crux#54]
Look at octoglow's Main.cx

**Comment by andyfriesen** [comment]
Specifically, the closing braces in this example cannot be dedented to the level of the `let`:

```
       let mediaType = if asset->endsWith(".html") {
           "text/html"
         } else if asset->endsWith(".css") {
           "text/css"
         } else if asset->endsWith(".js") {
           "text/javascript"
         } else {
           return errorResponse(response, 404, "Not found")
         }
```

**Comment by chadaustin** [comment]
Working on this now.

**Comment by chadaustin** [comment]
Fixed in f3a49c4 and related commits 4dd8164 and 216eecb4.
